bears/general/KeywordBear.py

```diff
@@ -18,8 +19,12 @@ def _get_comments(dependency_results):
             not isinstance(annotation_bear_results, list)):
         return
 
-    for result in annotation_bear_results:
-        yield from result.contents.get('comments', [])
+    try:
```

Comment:
why is this needed? If the comments are unavailable that should be atched by the ifs above